src/sidebar.js

```diff
@@ -35,7 +30,6 @@ const styles = {
     right: 0,
     bottom: 0,
     overflow: 'auto',
-    transition: 'left .3s ease-out',
```

Comment:
Instead of setting this attribute each render we can simply enable the transition for both `left` and `right` attribute by separating them with a comma:
```css
transition: left .3s ease-out, right .3s ease-out
```